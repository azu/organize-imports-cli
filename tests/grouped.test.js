import { test, expect } from '@jest/globals';
import runCli from "./run-cli.js";

test("grouped imports - processes grouped imports", async () => {
  const result = await runCli("cli/grouped", ["file.js"]);
  expect(result.status).toMatchSnapshot();
  expect(result.stdout).toMatchSnapshot();
  expect(result.stderr).toMatchSnapshot();
  expect(result.write).toMatchSnapshot();
});
