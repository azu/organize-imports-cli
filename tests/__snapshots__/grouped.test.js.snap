// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grouped imports - processes grouped imports 1`] = `1`;

exports[`grouped imports - processes grouped imports 2`] = `""`;

exports[`grouped imports - processes grouped imports 3`] = `"appendStdout is not defined"`;

exports[`grouped imports - processes grouped imports 4`] = `[]`;
